### DISCUSSION TITLE
Unable to resolve 'cake.tool (>= 1.1.0)' for 'net5.0'

### DISCUSSION THREAD

**Discussion by NathanGloyn** [cake-build/cake#3357] [Q&A] [answered]
I am trying to follow the "Getting started" instructions on the [website ](https://www.cakebuild.net/docs/getting-started/setting-up-a-new-project) but when I run `dotnet tool install Cake.Tool --version 1.1.0` I get the error in the title.

The full error is:

```
error NU1100: Unable to resolve 'cake.tool (>= 1.1.0)' for 'net5.0'.
error NU1100: Unable to resolve 'cake.tool (>= 1.1.0)' for 'net5.0/any'.
The tool package could not be restored.
Tool 'cake.tool' failed to install. This failure may have been caused by:

* You are attempting to install a preview release and did not use the --version option to specify the version.
* A package by this name was found, but it was not a .NET tool.
* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
* You mistyped the name of the tool.

For more reasons, including package naming enforcement, visit https://aka.ms/failure-installing-tool
```

Any ideas on how to move forward?

**Comment by devlead** [comment, marked as answer]
I just ran on Windows, MacOS and Linux
```
dotnet tool install Cake.Tool --version 1.1.0
```
and got no failures
```diff
+ You can invoke the tool from this directory using the following commands: 'dotnet tool run dotnet-cake' or 'dotnet dotnet-cake'.
+ Tool 'cake.tool' (version '1.1.0') was successfully installed. Entry is added to the manifest file [REDACTED]\.config\dotnet-tools.json.
```

* Which version of .NET SDK do you have installed? (believe 5.0.203 is latest - `dotnet --info` will tell you which you've got installed)
* Which operating system are you on?
* Which NuGet sources to you get if you execute `dotnet nuget list source` any non-standard there that could cause issues?
* Does it help if you've got an empty nuget.config in folder your running from (can be created using `dotnet new nugetconfig`)
* Does it help if you pin .NET SDK Version with global.json (can be created using `dotnet new globaljson`)

Running with detailed logging might give some  more clues
```bash
dotnet tool install Cake.Tool --version 1.1.0 --verbosity detailed
```

**Reply by NathanGloyn** [reply]
I started gathering the info from the various sources but it was when I did `dotnet nuget list source` that I found the problem. I have no idea why (new VM, newly installed Vs, etc) but the only source I had listed was `Microsoft Visual Studio Offline Packages` which completely explains why it didn't work. 

As soon as I added in Nuget it worked, using `dotnet` is new (been framework up to now) so TIL about nuget sources, thank you for that, another check to add to my VM setup

**Reply by devlead** [reply]
Glad you got it sorted 👍

**Reply by gep13** [reply]
I spoke with @NathanGloyn over Twitter DM's, and I am wondering if this issue was caused by this problem in Chocolatey (Nathan mentioned that he had used Chocolatey to provision the VM): https://github.com/chocolatey/choco/issues/2233 Basically, Chocolatey (for a LONG time) has been creating a empty NUGet.config file, that it itself doesn't use.  Something has changed in NuGet and by extension dotnet cli, where it now looks at this empty NuGet.config file, and then isn't able to restore packages.  This issue is going to be addressed in the next release of Chocolatey, but wanted to mention it here as well in case anyone else runs into this.
